newrelic_lambda_cli/templates/license-key-secret.yaml

```diff
@@ -51,3 +54,7 @@ Outputs:
     Value: !Ref ViewNewRelicLicenseKeyPolicy
     Export:
       Name: !Sub "${AWS::StackName}-${ViewPolicyExportName}"
+  NrAccountId:
+    Value: !Ref NrAccountId
+    Export:
+      Name: !Sub "${NrAccountId}"
```

Comment:
We should prefix the name with the stack name. The export namespace is flat, IIRC, and the stack name is guaranteed unique. Also, the name shouldn't include the value. So something like:

    !Sub "${AWS::StackName}-NrAccountId"

Should work well.

Since the CLI uses a consistent stack name, we can use the stack name to find the output value in the call to the `_get_cf_stack_status` function